docs/getting_started_guide/Installing_SIMP_using_r10k.rst

```diff
@@ -0,0 +1,111 @@
+tools, with a little tweaking.
+
+Read the introduction documentation on whichever of these technologies that is
+being used:
+ * Code Manager: https://docs.puppet.com/pe/latest/code_mgr.html
+ * r10k: https://github.com/puppetlabs/r10k/blob/master/README.mkd
+
+``r10k`` will be used to reference both ``r10k`` itself and ``Code Manager``
+throughout this document. This document will also assume the SIMP server has
+Internet access.
+
+If your system does not have Internet access, you will need to adjust any paths
+to point to your internal mirrors.
+
+.. NOTE::
+
+  If SIMP is not installed via ISO, disk partitioning is left to the user.
+  Make sure that your intended SIMP system meets guidelines for partitioning.
+
+Installation of r10k
+^^^^^^^^^^^^^^^^^^^^
+
+On the SIMP server, run the following command to install the ``r10k`` ruby gem
+into the vendor ruby that comes with the ``puppet-agent`` package:
+
+.. code-block:: bash
+
+  $ /opt/puppetlabs/puppet/bin/gem install r10k
+
+``r10k`` can be used by calling the absolute path of the executable:
+
+.. code-block:: bash
+
+  $ /opt/puppetlabs/puppet/bin/r10k help
+
+
+Setting up your Control Repo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SIMP uses a :term:`Puppetfile` and distributes a :term:`Control Repo` in the
+``simp-environment`` package, or also in the `simp-core`_ repository.
+
+On the filesystem of an installed SIMP system:
+
+.. code-block:: bash
+
+  $ tree -L 1 /etc/puppetlabs/code/environments/simp/
+  /etc/puppetlabs/code/environments/simp/
+  ├── environment.conf
+  ├── FakeCA
+  ├── hieradata
+  ├── manifests
+  ├── modules
+  ├── simp_autofiles
+  └── site_files
+
+Our git repo:
+
+.. code-block:: bash
+
+  $ tree -L 1 src/assets/simp-environment/environments/simp
+  src/assets/simp-environment/environments/simp
+  ├── environment.conf
+  ├── FakeCA/
+  ├── hieradata/
+  ├── manifests/
+  └── site_files/
+
+A Control Repo contains the modules, hieradata, and other manifests required for
+an infrastructure, and keeping it all in a git repo creates a workflow for
+updating and developing on your Puppet infrastructure.
+
+The modules are defined in a :term:`Puppetfile`, which SIMP uses as a development
+tool. We keep an up-to-date Puppetfile in the core of our repo, which you can
+download using this snippet:
+
+.. code-block:: bash
+
+  $ curl -o Puppetfile https://github.com/simp/simp-core/blob/master/Puppetfile.stable
+
+The example Puppetfile is labeled *stable*, meaning that the versions of the
+modules it contains are the ones contained in the last SIMP release. You can go
+to any previous release and download a Puppetfile with references to older
+modules from the git history of the ``simp-core`` repo.
+
+Our Puppetfile pulls down every dependency needed for SIMP builds, which
+contains more than just modules. Open up your ``Puppetfile`` and remove the
+lines from ``moduledir 'src'`` to ``moduledir 'src/puppet/modules'`` so that
```

Comment:
Need to tell users to keep the simp-rsync stuff but point it where it's supposed to go and then run the rsync.facl code.

Not *quite* sure how to do this.